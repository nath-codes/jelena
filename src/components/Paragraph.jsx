import styled from "styled-components";
import spacer from "../mixins/spacer";

const Paragraph = styled.p`
  margin: 0 0 ${props => spacer(1, props.theme.breakpoint)};

  &:last-child {
    margin-bottom: 0;
  }
`;
export default Paragraph;
