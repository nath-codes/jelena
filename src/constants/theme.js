export default {
  fonts: {
    heading: "bebas-neue",
    body: "montserrat"
  },
  colors: {
    delftBlue: "#1b56c5",
  }
};
