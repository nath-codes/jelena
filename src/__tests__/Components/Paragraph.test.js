import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import Paragraph from "../../components/Paragraph";
import theme from "../../constants/theme";

describe("<Paragraph />", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Paragraph theme={theme} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
