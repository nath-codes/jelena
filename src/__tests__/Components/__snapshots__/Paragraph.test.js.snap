// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Paragraph /> renders correctly 1`] = `
.c0 {
  margin: 0 0 1.2rem;
}

.c0:last-child {
  margin-bottom: 0;
}

<p
  className="c0"
/>
`;
